src/styles/block/starts.scss

```diff
@@ -0,0 +1,25 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  display: flex;
+  background-image: url(../images/star.svg);
+  background-position: center;
+  height: 16px;
+  width: 16px;
+  background-repeat: no-repeat;
+  margin-left: 4px;
+}
+
+.stars__star:first-child {
+  margin-left: 0;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 .stars__star:nth-child(-n + 5) {
```

Comment:
According to BEM naming convention, modifiers should be named with a single dash and a key-value pair. For example, instead of '.stars--1', it should be '.stars--stars-1'. This makes it clear that 'stars' is the key and '1' is the value of the modifier. Please rename the modifiers accordingly.